Formula/d/dotnet.rb

```diff
@@ -39,62 +36,30 @@ class Dotnet < Formula
   # GCC builds have limited support via community.
   fails_with :gcc
 
-  # Backport fix for error on aspnetcore version while building 'installer in tarball'.
-  # TODO: Remove when available in release.
-  # PR ref: https://github.com/dotnet/installer/pull/14938
-  patch do
-    url "https://raw.githubusercontent.com/Homebrew/formula-patches/f206f7a45b330cce79e6bfe9116fccd93b0d3ed8/dotnet/aspnetcore-version.patch"
-    sha256 "00103452e2f52831c04007f1b7f9fcd5ecddf0671943657104f0ac8d3a9ca613"
-  end
-
-  # Fix build failure on macOS due to missing bootstrap packages
-  # Fix build failure on macOS ARM due to `osx-x64` override
-  # Issue ref: https://github.com/dotnet/source-build/issues/2795
-  patch :DATA
-
   def install
-    if OS.linux?
-      ENV.append_path "LD_LIBRARY_PATH", Formula["icu4c"].opt_lib
-      ENV.append_to_cflags "-I#{Formula["krb5"].opt_include}"
-    end
```

Comment:
Probably need to re-add this for Linux (since we use non-standard paths) but will give run a try first without extra paths.